README.md

```diff
@@ -90,6 +91,15 @@ permissions:
   actions: read
 ```
 
+## Support for Github Enterprise
+
+To run this action in your Github Enterprise (GHE) instance you need to override `github-api-url`:
+
+```yaml
+with:
+  github-api-url: 'https://ghe-host.acme.net/api/v3'
```

Comment:
```suggestion
  github-api-url: 'https://ghe-host.example.net/api/v3'
```

I don't know the detail for acme and the domain owner, don't we need to use `example.*` for this use? https://www.rfc-editor.org/rfc/rfc2606.txt

Reply:
Ah, I guess resolving conflict drops old review points... https://github.com/kachick/wait-other-jobs/pull/618#discussion_r1365897262

Sorry for my lately response 🙇‍♂️